requirements.txt

```diff
@@ -48,3 +48,5 @@ einops
 transformers
 #deps for bark
 encodec
+#deps for Belarusian phonemizer
+jpype1
```

Comment:
This is a big dependency. It is better to keep out of the requirements but warn the user when the phonemizer is called without it being installed.

Reply:
Right. Fixed by change TTS/tts/utils/text/belarusian/phonemizer.py